es_loader/es_loader.py

```diff
@@ -41,12 +72,49 @@ def _create_index(self):
         self.es.indices.create(self.index_name)
         logger.debug('Index created')
 
+    def _get_job(self):
+        """Get job from ScrapingHubClient istance."""
+        self.job = self.sc.get_job(self.job_id)
+
+    def _get_job_metadata(self):
+        """Get job metadata."""
+        self.metadata = self.job.metadata.list()
```

Comment:
Same comment as above, for the _get_job method. Also `self.metadata` is only used in the `_get_scraped_items_count`, so maybe it's not worth putting it into `self`.